### PR TITLE
IIS Metrics

### PR DESCRIPTION
**Pull request by thiagoftsm**
This PR is adding metrics to monitor all IIS sites:

### CODE REVIEW

File: modules/wmi/charts.go [hunk 1]
```diff
@@ -598,6 +611,132 @@ var (
...
+		IISErrorChart.Copy(),
+	}
+	IISActiveConnectionChartsChart = module.Chart{
+		ID:       "iis_active_conn",
+		Title:    "Website Active connections",
+		Units:    "connections",
+		Fam:      "IIS",
+		Ctx:      "wmi.iis_active_conn",
+		Type:     module.Stacked,
+		Priority: prioIISActiveConns,
```
Comment (ilyam8) [inline modules/wmi/charts.go:637]: We need a chart per site, not a dimension per site.

**Review by ilyam8 — CHANGES_REQUESTED** [review]
a chart per site, please

**Comment by ilyam8** [comment]
@thiagoftsm on second thought, I have no problem with merging it as it is (a dimension per site). The just means we will need to change it in the future. So you can keep it as it is if you want.

**Comment by thiagoftsm** [comment]
> @thiagoftsm on second thought, I have no problem with merging it as it is (a dimension per site). The just means we will need to change it in the future. So you can keep it as it is if you want.

I am already working to modify it. Until the end of the day we will have per site. :handshake:

File: modules/wmi/charts.go (hunk 1 above)
Comment (thiagoftsm) [reply modules/wmi/charts.go]: Done

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**